src/Hive.Core/sia380/sia380.py

```diff
@@ -205,35 +206,43 @@ def main(room_properties, floor_area, T_e, setpoints_ub, setpoints_lb, surface_a
     # preAllocate arrays. its a bit faster than .append (https://levelup.gitconnected.com/faster-lists-in-python-4c4287502f0a)
     Q_i = [0.0] * months_per_year
     Q_s = [0.0] * months_per_year
-    Q_i_eta_g = [0.0] * months_per_year
-    Q_s_eta_g = [0.0] * months_per_year
-    Q_V = [0.0] * months_per_year
-    Q_T = [0.0] * months_per_year
-    QT_opaque = [0.0] * months_per_year
-    QT_transparent = [0.0] * months_per_year
+    Q_i_out = [0.0] * months_per_year
+    Q_s_out = [0.0] * months_per_year
+    Q_V_out = [0.0] * months_per_year
+    Q_T_out = [0.0] * months_per_year
+    QT_opaque_out = [0.0] * months_per_year
+    QT_transparent_out = [0.0] * months_per_year
 
     Q_Heat = [0.0] * months_per_year
     Q_Cool = [0.0] * months_per_year
     Q_Elec = [0.0] * months_per_year
 
-
     Phi_P_tot = Phi_P * floor_area
     Phi_L_tot = Phi_L * floor_area
     Phi_A_tot = Phi_A * floor_area
 
+    simulation_cases = 4
     # For each month, compute:
     for month in range(months_per_year):
         # External air flowrate (thermisch wirksamer Aussenluftvolumenstrom)
         Vdot_e = Vdot_e_spec * floor_area
         Vdot_inf = Vdot_inf_spec * floor_area
         Vdot_th = Vdot_e * (1 - eta_rec) + Vdot_inf
+        Vdot_th_no_heat_recovery = Vdot_e + Vdot_inf
 
         # Ventilation heat loss coefficient (Lüftungs-Wärmetransferkoeffizient), H_V
         H_V = Vdot_th/3600 * rho * c_p
+        H_V_no_heat_recovery = Vdot_th_no_heat_recovery/3600 * rho * c_p
 
         # Ventilation losses (Lüftungswärmeverluste), Q_V
-        Q_V_ub = H_V * (setpoints_ub[month] - T_e[month]) * t[month]
-        Q_V_lb = H_V * (setpoints_lb[month] - T_e[month]) * t[month]
+        Q_V_ub_heating = H_V * (setpoints_ub[month] - T_e[month]) * t[month]
```

Comment:
i'm _assuming_ `_ub_` and `_lb_` stand for "upper bound" and "lower bound" respectively. is that correct? maybe a quick comment for that?

Reply:
added comments